test/whitelisted_crowdsale.js

```diff
@@ -53,22 +46,62 @@ contract('WhitelistedCrowdsale', async (accounts) => {
       await instanceWhitelistedCrowdsale.endTime.call(), endTime
     );
     assert.equal(
-      await instanceWhitelistedCrowdsale.rate.call(), rate
+      await instanceWhitelistedCrowdsale.rate.call(), defaultRateForPrivateOffer
     );
     assert.equal(
-      await instanceWhitelistedCrowdsale.minPurchase.call(), minPurchase
+      await instanceWhitelistedCrowdsale.minPurchase.call(), defaultMinPurchaseForPrivateOffer
     );
     assert.equal(
-      await instanceWhitelistedCrowdsale.maxPurchase.call(), maxPurchase
+      await instanceWhitelistedCrowdsale.maxPurchase.call(), defaultMaxPurchaseForPrivateOffer
     );
     assert.equal(
-      await instanceWhitelistedCrowdsale.wallet.call(), wallet
+      await instanceWhitelistedCrowdsale.wallet.call(), escrowAccount
     );
     assert.equal(
       await instanceWhitelistedCrowdsale.token.call(), token.address
     );
     assert.equal(
-      await instanceWhitelistedCrowdsale.countPurchaseAmount.call(), countPurchaseAmount
+      await instanceWhitelistedCrowdsale.countPurchaseAmount.call(), defaultCountPurchaseAmountForPrivateOffer
+    );
+  });
+
+  it("should throw error when the beneficiary not in whitelist", async () => {
+    try {
+      await instanceWhitelistedCrowdsale.buyTokens.sendTransaction(userNotFromWhitelist, {from: owner, value: web3.toWei(2, "ether")});
+      assert.ifError('Error, the owner should not be able to call this method');
+    } catch (err) {
+      assert.equal(err, 'Error: VM Exception while processing transaction: revert', "Benefeciary not in whitelist trying to buy tokens");
+    }
+  });
+
+  it("should throw error when the beneficiary has a zero address", async () => {
```

Comment:
I think it is better to say `should throw an error when the beneficiary is the contract owner`.